### DISCUSSION TITLE
Setting up Chandler Behind Cloudflare Tunnels?

### DISCUSSION THREAD

**Discussion by stanleyshly** [monicahq/monica#7112] [Q&A]
I just set up chandler and I'm accessing it on the LAN directly through the IP address, and it's working great. However, when using cloudflare tunnels and accessing the dashboard through the sign in page and sign in, the login page just comes back, with my filled in username and password gone. I setting the variable `APP_URL=monica.stanleyshen.org`, but that doesn't change anything. Does anyone know how to setup up chandler to work properly behind cloudflare tunnels?

Here is my current install method:
```
mysqlCid="$(docker run -d \
 -e MYSQL_ROOT_PASSWORD=sekret \
 -e MYSQL_DATABASE=monica \
 -e MYSQL_USER=homestead \
 -e MYSQL_PASSWORD=secret \
 "mysql:5.7")"
docker run -d \
 --link "$mysqlCid":mysql \
 -e DB_HOST=mysql \
 -e APP_URL=fqdn.url \
 -e APP_ENV=production  \
 -p 80:80 \
 ghcr.io/monicahq/monica-next:main
 

```

**Comment by brucealdridge** [comment]
I've created an issue for this https://github.com/monicahq/monica/issues/7120

**Reply by slifty** [reply]
@stanleyshly does the resolution in the above linked issue address your question (it looks like the solution there was to set `APP_TRUSTED_PROXIES=*`)

**Reply by stanleyshly** [reply]
@slifty Ill give it a try once I get back to my PC. I noticed in the config, you have it setup with docket with a persistent SQLite database. Would you be able to share your compose and/or docker commands so I can replicate your environment as closely as possible? I haven’t gotten a persistent DB working yet with chandler.

**Reply by slifty** [reply]
@stanleyshly regarding the persistent DB if it's OK I'd suggest redirecting over to this thread: https://github.com/monicahq/monica/discussions/7151#discussioncomment-8584833 -- I link to my docker compose over there!
